addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2869,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
```

Comment:
"SMB" is not a setting or a button. The message is for a setting, but SMB itself is a library inside of Kodi.